spec/modules/facts/plans/info_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt_spec/plans'
+
+describe 'facts::info' do
```

Comment:
We should be able to move this to the module tests since it mocks out the task running. 
There may be some work to do to expose the plan mock framework for those tests though